### PR TITLE
Update conf.py

### PR DESCRIPTION
**Pull request by FinalAngel**
- rename to "Aldryn News & Blog"
- add "Divio AG" as author

### CODE REVIEW

**Comment by FinalAngel** [comment]
@mkoistinen if it is good, you can merge! I'm offline till later tonight.

File: docs/conf.py [hunk 1]
```diff
@@ -43,17 +43,17 @@
...
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.1'
+version = '1.0.0'
 # The full version, including alpha/beta/rc tags.
-release = '0.0.1'
+release = '1.0.0'
```
Comment (mkoistinen) [inline docs/conf.py]: Is it possible to reference the version in __init__.py rather than bake it in here? Genuine question.

**Comment by FinalAngel** [comment]
@mkoistinen I adapted conf.py to take the version from aldryn_newsblog, nice call :)

**Comment by FinalAngel** [comment]
@mkoistinen the build fails on ``flake8`` https://travis-ci.org/aldryn/aldryn-newsblog/jobs/67794822 dunno if that's a known issue.

**Comment by intellectronica** [comment]
:+1:

File: docs/conf.py [hunk 2]
```diff
@@ -251,8 +252,8 @@
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [(
-    'index', 'AldrynNewsblog', u'Aldryn Newsblog Documentation',
-    u'Tom Berger', 'AldrynNewsblog', 'One line description of project.',
+    'index', 'AldrynNewsblog', u'Aldryn News & Blog Documentation',
+    u'Divio AG', 'AldrynNewsblog', 'Aldryn News & Blog is an Aldryn-compatible news and weblog application for django CMS.',
```
Comment (intellectronica) [inline docs/conf.py]: This line is too long and fails lint. Easy to fix - break it down like this:
```
u'Divio AG', 'AldrynNewsblog', (
    'Aldryn News & Blog is an Aldryn-compatible news and weblog'
    ' application for django CMS.'),
```

**Pull request merged**